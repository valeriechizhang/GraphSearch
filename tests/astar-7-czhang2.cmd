--energy 500 --end-x=0  --end-y=1