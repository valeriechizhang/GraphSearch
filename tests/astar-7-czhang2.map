84  24  90  7   67  6   132
86  13  89  67  62  32  7
54  67  88  66  22  13  12
18  17  43  13  73  10  62
19  16  15  76  22  43  6
20  50  14  13  12  19  98
21  51  23  95  11  100 52
22  68  45  74  10  9   8
1   2   3   4   5   6   7