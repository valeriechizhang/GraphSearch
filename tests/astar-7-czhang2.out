Solution steps: ['E', 'E', 'E', 'E', 'E', 'E', 'N', 'W', 'W', 'N', 'N', 'W', 'W', 'N', 'W', 'N', 'W', 'S', 'S', 'S', 'S']
Solution cost: 42
Number of states considered: 22

Frontier:
Pos=(4, 4) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'N', 'W', 'W', 'N', 'N', 'N'] Cost=123
Pos=(2, 2) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'N', 'W', 'W', 'N', 'N', 'W', 'W', 'S'] Cost=108
Pos=(5, 3) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'N', 'W', 'W', 'N', 'N', 'E'] Cost=72

Closed List:
Pos=(0, 0) Moves=[] Cost=0
Pos=(1, 0) Moves=['E'] Cost=2
Pos=(2, 0) Moves=['E', 'E'] Cost=4
Pos=(3, 0) Moves=['E', 'E', 'E'] Cost=6
Pos=(4, 0) Moves=['E', 'E', 'E', 'E'] Cost=8
Pos=(5, 0) Moves=['E', 'E', 'E', 'E', 'E'] Cost=10
Pos=(6, 0) Moves=['E', 'E', 'E', 'E', 'E', 'E'] Cost=12
Pos=(6, 1) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'N'] Cost=14
Pos=(5, 1) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'N', 'W'] Cost=16
Pos=(4, 1) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'N', 'W', 'W'] Cost=18
Pos=(4, 2) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'N', 'W', 'W', 'N'] Cost=20
Pos=(4, 3) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'N', 'W', 'W', 'N', 'N'] Cost=22
Pos=(3, 3) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'N', 'W', 'W', 'N', 'N', 'W'] Cost=24
Pos=(2, 3) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'N', 'W', 'W', 'N', 'N', 'W', 'W'] Cost=26
Pos=(2, 4) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'N', 'W', 'W', 'N', 'N', 'W', 'W', 'N'] Cost=28
Pos=(1, 4) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'N', 'W', 'W', 'N', 'N', 'W', 'W', 'N', 'W'] Cost=30
Pos=(1, 5) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'N', 'W', 'W', 'N', 'N', 'W', 'W', 'N', 'W', 'N'] Cost=32
Pos=(0, 5) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'N', 'W', 'W', 'N', 'N', 'W', 'W', 'N', 'W', 'N', 'W'] Cost=34
Pos=(0, 4) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'N', 'W', 'W', 'N', 'N', 'W', 'W', 'N', 'W', 'N', 'W', 'S'] Cost=36
Pos=(0, 3) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'N', 'W', 'W', 'N', 'N', 'W', 'W', 'N', 'W', 'N', 'W', 'S', 'S'] Cost=38
Pos=(0, 2) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'N', 'W', 'W', 'N', 'N', 'W', 'W', 'N', 'W', 'N', 'W', 'S', 'S', 'S'] Cost=40
Pos=(0, 1) Moves=['E', 'E', 'E', 'E', 'E', 'E', 'N', 'W', 'W', 'N', 'N', 'W', 'W', 'N', 'W', 'N', 'W', 'S', 'S', 'S', 'S'] Cost=42
